test/test_test_utils.py

```diff
@@ -46,11 +45,7 @@ def test_get_log_for_test(self):
     with open(os.path.join(output_path, 'test.log'), 'w') as f:
       f.write('23:59:59 [main] INFO  Testing_Logs')      
     with open(os.path.join(output_path, 'test.log'), 'w') as f:
-      f.write('23:59:59 [main] INFO  TestClientService - Sent 100')
-
-    log_file = os.path.join(output_path, "test.log")
```

Comment:
Not related to travis.. This was a new test added for coverage and had some issues in it.